devlib/target.py

```diff
@@ -945,10 +945,7 @@ def capture_screen(self, filepath):
 
     def _resolve_paths(self):
         if self.working_directory is None:
-            if self.connected_as_root:
-                self.working_directory = '/root/devlib-target'
-            else:
-                self.working_directory = '/home/{}/devlib-target'.format(self.user)
+            self.working_directory = self.execute("awk -F: '/{}/ {{ print $6 }}' /etc/passwd".format(self.user)).strip()
```

Comment:
`awk` must be invoked via `self.busybox`.

Reply:
i do think we're ok relying on awk being present and we can't use busybox because it hasn't been installed yet.

i think there is a more straightforward solution, though: how about using the output of ``` `pwd` ``` or  ```echo $PWD``` ? the really nice thing about this is that it should fall back gracefully, even if someone didn't create ~user or authentication doesn't actually work via /etc/passwd.

will push the change accordingly, please let me know what you think.